crates/uv-workspace/src/pyproject_mut.rs

```diff
@@ -35,13 +37,18 @@ pub enum Error {
 }
 
 impl PyProjectTomlMut {
-    /// Initialize a `PyProjectTomlMut` from a `PyProjectToml`.
+    /// Initialize a [`PyProjectTomlMut`] from a [`PyProjectToml`].
     pub fn from_toml(pyproject: &PyProjectToml) -> Result<Self, Error> {
         Ok(Self {
             doc: pyproject.raw.parse().map_err(Box::new)?,
         })
     }
 
+    /// Initialize a [`PyProjectToml`] from a [`PyProjectTomlMut`].
+    pub fn to_toml(&self) -> Result<PyProjectToml, Error> {
+        Ok(toml::from_str(&self.doc.to_string()).map_err(Box::new)?)
```

Comment:
I feel like there should be a more efficient way to do this? We don't even need the full-fidelity string here.